step-function/get-files/src/main.py

```diff
@@ -0,0 +1,24 @@
+from os import environ
+from os.path import basename
+
+import boto3
+
+S3_CLIENT = boto3.client('s3')
+
+
+def get_download_url(key):
+    bucket = environ['BUCKET']
+    region = environ['AWS_REGION']
+    return f'https://{bucket}.s3.{region}.amazonaws.com/{key}'
+
+
+def lambda_handler(event, context):
+    objects = S3_CLIENT.list_objects_v2(Bucket=environ['BUCKET'], Prefix=event['job_id'])
```

Comment:
think I like `response` better than `objects`, since this is a dictionary and not (yet) a list of s3 objects